funnel/models/event.py

```diff
@@ -52,6 +53,25 @@ class Event(BaseMixin, db.Model):
     ticket_types = db.relationship("TicketType", secondary=event_ticket_type, lazy='dynamic')
     participants = association_proxy('attendees', 'participant')
 
+    @classmethod
+    def get_or_create(cls, name, space):
+        event = cls.query.filter_by(name=name, proposal_space=space).first()
+        if not event:
+            event = cls(name=name, proposal_space=space)
+            db.session.add(event)
+            db.session.commit()
+        return event
+
+    @classmethod
+    def sync_from_list(cls, event_list, space):
+        for event_dict in event_list:
+            event = cls.get_or_create(event_dict.get('name').decode('utf-8'), space)
+            for ticket_type_name in event_dict.get('ticket_types', []):
+                if ticket_type_name.decode('utf-8') not in [ticket_type.name for ticket_type in event.ticket_types]:
+                    event.ticket_types.append(TicketType.get_or_create(ticket_type_name.decode('utf-8'), space))
+        db.session.commit()
+        return True
```

Comment:
If it always returns `True`, returning any value is pointless. Skip the `return` statement.